plutus-ir/src/Language/PlutusIR/Analysis/Dependencies.hs

```diff
@@ -124,7 +126,7 @@ termDeps = \case
     Let _ _ bs t -> do
         bGraphs <- traverse bindingDeps bs
         bodyGraph <- termDeps t
-        pure $ G.overlays $ bGraphs ++ [bodyGraph]
+        pure . G.overlays . NE.toList $ bodyGraph NE.<| bGraphs
```

Comment:
I changed this to append the bodygraph in the front (for efficiency). I think the order does not matter.